scan/action.yml

```diff
@@ -77,14 +74,23 @@ runs:
           console.log(`SARIF file name: ${sarifFilePath}`);          
           return sarifFilePath;
 
+    # Annotate the SARIF file tags to include the monorepo's current project name
     - name: Annotate CodeQL Alert SARIF with Project tag
-      if: matrix.project.build_mode != 'other'  
-      uses: advanced-security/monorepo-code-scanning-action/sarif-project-annotator@annotate-sarif
```

Comment:
This was the major change, to no longer have to depend on internal versions... instead reference the script directly

```diff
+const script = require(path.join(action_path, '..', 'sarif-project-annotator', 'sarif-project-annotator.js'))
```